styles.css

```diff
@@ -3,4 +3,110 @@ body{
 }
 h1{
     color:#6b37c1;
+}
+#services{
+    box-sizing: border-box; 
+    /*min-height: 100vh;*/
+    /* max-width: 250px; */
+    padding: 20px;
+    background-color:#bae1ff;
+}
+.flip-container{
+    perspective: 1000px;
+    margin: 3%;
+}
+.front-card-creativity h2 info-title{
+    margin: 0%;
+    font-family: Arial, Helvetica, sans-serif;
+    text-align: center;
+}
+
+.front-card-creativity{
+   font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
```

Comment:
For our website to be consistent, I'd suggest we decide on the main font and we use it for the entire page rather than having different fonts in different sections :) 
What do you think? When we decide on the font, we can add it to the root.

Reply:
Agree completely, once we choose a font it'll be consistent